i18n/fr.toml

```diff
@@ -4,19 +4,19 @@
 
 # === init ===
 [configurationError]
-other = "" # waitting for translation
+other = "Erreur de configuration\nVous n'avez pas configuré la version de FixIt correctement. Voir https://fixit.lruihao.cn/theme-documentation-basics/#basic-configuration"
 
 [compatibilityError]
-other = "" # waitting for translation
+other = "Erreur de compatibilité ({{ .From }} -> {{ .To }}):\Vous avez une mise à jour incompatible. Voir https://github.com/hugo-fixit/FixIt/releases"
```

Comment:
This line has format error, and it should be:

```toml
other = "Erreur de compatibilité ({{ .From }} -> {{ .To }}):\nVous avez une mise à jour incompatible. Voir https://github.com/hugo-fixit/FixIt/releases"
```

Reply:
Oops, sorry. I have fixed it, I will let you confirm and merge if everything is ok.